client/components/RatingsAndReviews/Reviews/Review.jsx

```diff
@@ -63,6 +63,9 @@ const Review = ({ review }) => {
         <Grid item xs={12}>
           <Divider className={classes.dividerLine} />
         </Grid>
+        <Grid item xs={12}>
+          {/* for spacing purposes */}
```

Comment:
Delete comment.

Reply:
This comment explains to someone reading the code, why there is an empty Grid item here 😆 Removing it messes up the alignment and I'm not sure how else to add spacing between two grid items. 
Also commenting the code to improve it's readability I think is okay

Reply:
Sounds good!